Cargo.toml

```diff
@@ -15,14 +15,15 @@ all-features = true
 maintenance = { status = "actively-developed" }
 
 [features]
-default = ["reqwest", "rustls-tls"]
+default = ["reqwest", "rustls-tls", "backchannel-logout"]
 curl = ["oauth2/curl"]
 reqwest = ["oauth2/reqwest"]
 ureq = ["oauth2/ureq"]
 native-tls = ["oauth2/native-tls"]
 rustls-tls = ["oauth2/rustls-tls"]
 accept-rfc3339-timestamps = []
 nightly = []
+backchannel-logout = []
```

Comment:
I just thought that since it is still a draft and breaking changes could happen, this way, there's no need for a major bump if we change something there (according to semver)

Reply:
ah gotcha. I think I'd prefer to have a full working implementation in this PR before merging it so that it's ready to release at that point

Reply:
Well that would depend on the OpenID working group to finalise the specification